.github/workflows/kuttl_tests.yaml

```diff
@@ -75,6 +75,17 @@ jobs:
     steps:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
+    - name: Set up Go 1.16
```

Comment:
Thanks for identifying and fixing this @burmanm, I can see what you meant about these tests using the wrong Go version now.